README.md

```diff
@@ -62,3 +63,15 @@ The names and definitions of the fixed properties that form the stamp descriptor
 * `staticProperties` - A set of static properties that will be copied by assignment to the stamp.
 * `propertyDescriptors` - A set of [object property descriptors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties) used for fine-grained control over object property behaviors
 * `configuration` - A set of options made available to the stamp and its initializers during object instance creation. Configuration properties get deep merged.
+
+#### Combining descriptors
```

Comment:
I don't have a strong opinion. If you think "compose" is better then okay.

I jsut thought that we are **composing stamps** but **mixing descriptos**. This will be less confusing.